nautobot.code-workspace

```diff
@@ -26,9 +20,6 @@
 		"[javascript]": {
 			"editor.formatOnSave": true
 		},
-		"[javascriptreact]": {
-			"editor.formatOnSave": true
-		},
 		"[jsx-tags]": {
 			"editor.formatOnSave": true
 		}
```

Comment:
Do we still need the `jsx-tags` section? I don't think this is even valid because `jsx-tags` is an embedded language inside javascript and javascriptreact.